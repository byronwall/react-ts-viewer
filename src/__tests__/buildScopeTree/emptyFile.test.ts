import { describe, it } from "vitest";
import { runScopeTreeSnapshotTest } from "./testUtils";

describe("buildScopeTree - Empty File", () => {
  it("should handle an empty file", () => {
    runScopeTreeSnapshotTest({
      snapshotIdentifier: "emptyFile",
      filePath: "empty.tsx", // This is the mock file path passed to buildScopeTree
      isFixture: false,
      inlineContent: "",
    });
  });
});
