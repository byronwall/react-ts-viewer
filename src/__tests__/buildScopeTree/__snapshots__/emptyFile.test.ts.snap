// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`buildScopeTree - Empty File > should handle an empty file > emptyFile 1`] = `
{
  "category": "Program",
  "children": [],
  "id": "empty.tsx",
  "kind": 307,
  "label": "empty.tsx",
  "loc": {
    "end": {
      "column": 0,
      "line": 1,
    },
    "start": {
      "column": 0,
      "line": 1,
    },
  },
  "source": "",
  "value": 0,
}
`;
